packages/@react-aria/focus/src/useFocusable.tsx

```diff
@@ -73,10 +74,10 @@ export function useFocusable(props: FocusableOptions, domRef: RefObject<HTMLElem
 
   useEffect(() => {
     if (autoFocusRef.current && domRef.current) {
-      domRef.current.focus();
+      focusSafely(domRef.current);
```

Comment:
Use focusSafely, so auto-focusing an element in a popover does not cause the popover to immediately close.